packages/clay-drop-down/src/Menu.tsx

```diff
@@ -254,20 +254,22 @@ const ClayDropDownMenu = React.forwardRef<HTMLDivElement, IProps>(
 					);
 				}
 
-				domAlign(
-					(ref as React.RefObject<HTMLElement>).current!,
-					alignElementRef.current,
-					{
-						offset: offsetFn(points),
-						overflow: {
-							adjustX: autoBestAlign,
-							adjustY: autoBestAlign,
-						},
-						points,
-					}
-				);
+				if ((ref as React.RefObject<HTMLElement>).current) {
```

Comment:
So this is what you mention in the commit message. Is the removal of `useCallback` also a required part of the fix? Or unrelated?

Reply:
The check before calling domAlign was the necessary part to fix the bug.

Removing `useCallback` was motivated more out of simplicity since `align` really isn't expensive or necessary to be memoized. I initially thought `useCallback`'s dependencies might be the bug, which led me to realize it was probably an over-complication of the code.

Reply:
Cool. Next time, that should go in the commit message (or better still, be a separate `refactor:` commit).

But overall lg2m.

![kim-jong-approval](https://user-images.githubusercontent.com/7074/109280720-ae616280-781b-11eb-9776-1404503b34aa.gif)